test/e2e/data/infrastructure-aws/kustomize_sources/limit-az/kustomization.yaml

```diff
@@ -2,3 +2,4 @@ resources:
   - ../default
 patchesStrategicMerge:
   - patches/limit-az.yaml
+  - patches/elb-health-check-protocol.yaml
```

Comment:
@sedefsavas Added a patch for custom protocol in existing test as we cannot reproduce it in upgrade test in remote mgmt cluster flavor.